sample/src/debug/java/com/polidea/cockpit/sample/apipresentation/ApiPresentationContract.kt

```diff
@@ -0,0 +1,18 @@
+package com.polidea.cockpit.sample.apipresentation
+
+interface ApiPresentationContract {
+
+    interface View : ApiPresentationBaseContract.View<Presenter> {
+        fun setDebugDescription(description: String)
+
+        fun showCockpitUi()
+    }
+
+    interface Presenter : ApiPresentationBaseContract.Presenter {
+        fun editValues()
+
+        fun setOnChangeListeners()
```

Comment:
This method is called inside `Presenter` instances only. Thus we don't have to expose it to `Presenter` interface. It's never not called from `View` instances.